cmd/push.go

```diff
@@ -125,17 +204,23 @@ func (p *PushCmdParams) validURL(s string) error {
 
 func (p *PushCmdParams) PreInteractive(ctx ActionCtx) error {
 	var err error
-	if !p.allAccounts {
+	if !p.allAccounts && !p.prune {
```

Comment:
How does prune work really - prune can only determine things based on locally available content). 
How does this work for NGS? (not from the user, but what if someone with permissions issued the prune there)

Reply:
This works by remotely deleting all jwt that do exist in the server but not locally.
Since the operator has to sign accounts he has them all.

For NGS we won't be using the nats based resolver.
When we do, only we do have permission to do so.
When we do, we have all accounts at our disposal.

Reply:
So currently in NGS we resolve via https. But my guess is that when this becomes available we would likely do updates via the system account. This means that if you have system account access, you could hose everything, even if normal submissions go through some other endpoint. The problem is the same with any other tech, but because `--prune` makes it so easy, it is very possible to create a problem.

Reply:
@kozlovic what do you think?